src/sentry/static/sentry/app/components/selectInput.jsx

```diff
@@ -2,6 +2,15 @@ import React from 'react';
 import jQuery from 'jquery';
 
 const SelectInput = React.createClass({
+  propTypes: {
+    disabled: React.PropTypes.bool,
+    multiple: React.PropTypes.bool,
+    required: React.PropTypes.bool,
+    placeholder: React.PropTypes.string,
+    value: React.PropTypes.string,
+    onChange: React.PropTypes.func,
```

Comment:
trailing comma or no trailing comma? Not sure if you had a rule here yet or a preference. Seems that you generally don't have one.